library/std/src/panic.rs

```diff
@@ -183,6 +183,10 @@ pub fn resume_unwind(payload: Box<dyn Any + Send>) -> ! {
 /// and if there is a panic, the abort will occur without allocating provided that the arguments to
 /// panic can be formatted without allocating.
 ///
+/// When using this function on a target which only supports thread local storage through the OS and
```

Comment:
I don't think we should document this behavior -- it's true, but it's really documenting a limitation we have in an extremely specific edge-case, and in principal the details of this limitation could change. It's hard to imagine how users would benefit from being able to rely on this anyway.

Reply:
Agreed and removed. 👍